src/pages/Admin/Charity/EditProfile/Slug.tsx

```diff
@@ -1,8 +1,13 @@
 import { Field } from "components/form";
 import { BASE_URL } from "constants/env";
+import { appRoutes } from "constants/routes";
+import { Link } from "react-router-dom";
 import { FV } from "./types";
 
-export default function Slug() {
+export default function Slug({ initSlug = "" }) {
+  const link =
+    initSlug && `${window.origin}${appRoutes.marketplace}/${initSlug}`;
```

Comment:
Judging by the wording of the Linear ticket, I think the requirement was to show what the slug would look like as the input value is changed, examples below:
*user starts typing*
slug: `e`
Current URL: `http://localhost:4200/marketplace/e`
slug: `ex`
Current URL: `http://localhost:4200/marketplace/ex`
slug: `exa`
Current URL: `http://localhost:4200/marketplace/exa`
slug: `example`
Current URL: `http://localhost:4200/marketplace/example`

I.e. show in the URL whatever is in the slug input field.

Wasn't this the idea?

Reply:
@ap-justin @misicnenad Ah, english... nebulous, unlike computer code 😆 In my head, I thought of it as applying to submitted, existing slugs only:
- IF `slug != ""` >> show current slug link
- ELSE >> show nothing there. Help text would start where the link sit.

 When a user opens that page, and the slug was set they see the link, if not set no link. After submission, when slug is set, can show the link.

Reply:
@SovereignAndrey, @misicnenad, implemented live example 
no slug provided
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/c6afec9b-af6c-4e9c-82bb-c49084d852b8)

slug provided
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/1131efad-6692-47ca-87fd-780d710d5691)

Reply:
@ap-justin After playing with the live version a bit I'd say let's revert that commit (4f408404) and just go with how it was. Tim can provide more clarity as he gets feedback from nonprofits using the form if there's still confusion or ways to improve it. 👍

Reply:
@ap-justin After playing with the live version a bit I'd say let's revert that commit (4f408404) and just go with how it was. Tim can provide more clarity as he gets feedback from nonprofits using the form if there's still confusion or ways to improve it. 👍